index.html

```diff
@@ -116,7 +116,7 @@ <h3 class="headline">Here's a time line of Dr. Borlaug's life:</h3>
         </blockquote>
     </div>
     </div>>
-      <footer class= "footer" >
+     > <footer class= "footer" >
```

Comment:
Do not add blank lines, spaces, or indentations without a reason.[]
kindly remove the space after the equal sign.